Test-set for service qa-scooter

itbel4onok opened the pull request:
Created test-set for service qa-scooter

itbel4onok committed:
Test-set for service qa-scooter

nikolay-977 commented on src/main/java/PageObject/MainPage/MainPage.java:
```diff
@@ -0,0 +1,108 @@
+package PageObject.MainPage;
```
Необходимо исправить наименование пакетов в соответствии с правилами нейминга. Пакеты — допустимы только строчные буквы. В случае, если слов несколько, их следует разделять точкой. Префикс должен быть одним из доменных имен верхнего уровня, таких как com, ru и т.д. Наименование пакета должно отражать суть содержимого.

nikolay-977 commented on src/test/java/Scooter/MainPage/QuestionPartTest.java:
```diff
@@ -0,0 +1,57 @@
+        this.expectedAnswerTextForQuestion = expectedAnswerTextForQuestion;
+    }
+
+    @Parameterized.Parameters
```
Можно улучшить: В параметризованных тестах для аннотации `@Parameterized.Parameters` лучше использовать аргумент name: `@Parameterized.Parameters(name = "Стоимость булочки. Тестовые данные: {0} {1}")`, где {0}, {1} - индексы параметров. С аргументом name наименование теста отобразится в отчете. А использование индексов параметров повысит информативность проверки.

itbel4onok committed:
Test-set for service qa-scooter
- packages names were changes
- the "name" parameter for parameterized tests has been added

nikolay-977 commented on README.md:
```diff
@@ -1 +1,23 @@
-# Sprint_4
\ No newline at end of file
+# Sprint_4
+
```
Необходимо исправить: добавь pom.xml.

itbel4onok committed:
Test-set for service qa-scooter
- packages names were changes
- the "name" parameter for parameterized tests has been added
- added pom.xml

nikolay-977 commented on pom.xml:
```diff
@@ -0,0 +1,33 @@
+
+    <!--Этот блок нужен для корректной работы с JDK9+-->
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
```
Необходимо исправить: в проекте следует использовать Java 11

nikolay-977 commented on src/main/java/ru/praktikum_services/qa_scooter/objects/order/OrderPageInitiallyForm.java:
```diff
@@ -0,0 +1,111 @@
+package ru.praktikum_services.qa_scooter.objects.order;
```
Необходимо исправить наименование пакетов в соответствии с правилами нейминга. Пакеты — допустимы только строчные буквы. **В случае, если слов несколько, их следует разделять точкой.** Префикс должен быть одним из доменных имен верхнего уровня, таких как com, ru и т.д. Наименование пакета должно отражать суть содержимого.
